tools/StaticAnalysis/Exceptions/AzureRM.Network/SignatureIssues.csv

```diff
@@ -249,6 +249,9 @@
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.SetAzureVirtualNetworkSubnetConfigCommand","Set-AzureRmVirtualNetworkSubnetConfig","1","8100","Set-AzureRmVirtualNetworkSubnetConfig Does not support ShouldProcess but the cmdlet verb Set indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.AddAzureVirtualNetworkSubnetConfigCommand","Add-AzureRmVirtualNetworkSubnetConfig","1","8100","Add-AzureRmVirtualNetworkSubnetConfig Does not support ShouldProcess but the cmdlet verb Add indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureVirtualNetworkSubnetConfigCommand","New-AzureRmVirtualNetworkSubnetConfig","1","8100","New-AzureRmVirtualNetworkSubnetConfig Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Network\Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureDelegationCommand","New-AzureRmDelegation","1","8100","New-AzureRmDelegation Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```

Comment:
you need to add these here as well : https://github.com/Azure/azure-powershell/tree/preview/tools/StaticAnalysis/Exceptions/Az.Network